### PR TITLE
doc: update README for v1.1.0 release[CI SKIP]

### PR DESCRIPTION
**Pull request by guangbochen**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Update README file for v1.1.0 release update

### CODE REVIEW

**Review by bk201 — APPROVED** [review]
LGTM!

**Review by Vicente-Cheng — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -13,36 +13,36 @@ Harvester is an open source [hyper-converged infrastructure](https://en.wikipedi
...
-| Memory | 32 GB minimum, 64 GB or above preferred |
-| Disk Capacity |  140 GB minimum, 500 GB or above preferred |
-| Disk Performance |  5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd). |
-| Network Card | 1 Gbps Ethernet minimum, 10Gbps Ethernet recommended |
-| Network Switch | Trunking of ports required for VLAN support |
+| CPU | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above preferred for production                                                                |
+| Memory | 32 GB minimum; 64 GB or above preferred                                                                                                                                                                    |
+| Disk Capacity | 140 GB minimum for testing; 500 GB or above preferred for production                                                                                                                                       |
```
Comment (bk201) [inline README.md]: 200 GB minimum is required for testing upgrade. Do we want to update here?

Comment (guangbochen) [reply README.md]: done, thanks.

**Pull request merged**